Add new product "jamfpro.md"

EdwinThomaz opened the pull request:
- Add new product "jamf pro"
- Four latest versions are added with end of support

EdwinThomaz committed:
Create Jamfpro.md

- Add new product "Jamf pro"

EdwinThomaz committed:
Update and rename Jamfpro.md to Jamfpro

EdwinThomaz reviewed (commented):
- Updated file "Jamfpro.md"

EdwinThomaz committed:
Update Jamfpro

- Updated release cycle format 
- Updated rom "Jamf pro 10.50.0" to "10.50.0"

EdwinThomaz reviewed (commented):
- Updated release cycle format
- Updated from "Jamf pro 10.50.0" to "10.50.0"

EdwinThomaz reviewed (commented):
- Updated file format

EdwinThomaz committed:
Update Jamfpro

- Droped activeSupportColumn and used the eolColumn
- Replaced support with eol for all releases
- disabled the the releaseColumn: false.

EdwinThomaz committed:
Update Jamfpro

- ReleaseDateColumn :true 
- Provided latest versions (with their latestReleaseDate) for each cycle

EdwinThomaz committed:
Update Jamfpro

- Updated header

EdwinThomaz committed:
Update Jamfpro

- Updated "Releasedatelatest" to "latest"

EdwinThomaz committed:
Update Jamfpro

- Updated releaseDateColumn: latest

EdwinThomaz committed:
Update Jamfpro

- releaseDateColumn: Released

EdwinThomaz committed:
Update Jamfpro

- Updated releaseCycle: to latest

EdwinThomaz committed:
Update Jamfpro

-Release date ; false

EdwinThomaz committed:
Update Jamfpro

EdwinThomaz committed:
Update and rename Jamfpro to Jamfpro.md

- Added releaseDate

EdwinThomaz committed:
Rename Jamfpro.md to Jamfpro

- Removed .md extension

EdwinThomaz committed:
Update Jamfpro

- Set releaseCycleColumn: true

EdwinThomaz committed:
Update Jamfpro

- Corrected format "releaseDate:"

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+---
+
```
```suggestion
```

Empty line is not needed.

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+---
+
+title: Jamf-Pro
```
```suggestion
title: Jamf Pro
```

The name used on https://www.jamf.com/products/jamf-pro/ does not contain a dash.

marcwrobel commented on products/Jamfpro:
The filename must be `jamf-pro.md`. Without the `.md` suffix the file will not be rendered properly.

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+
+title: Jamf-Pro
+category: device
+iconSlug: jamf_pro
```
```suggestion
```

There is no icon for this production on https://simpleicons.org/?q=jamf.

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+iconSlug: jamf_pro
+permalink: /jamf-pro
+versionCommand: jamf-pro --version
+releasePolicyLink: [https://www.jamf.com/resources/product-documentation/jamf-pro-release-notes/]
```
This link should point to the vendor documentation explaining how the product is released and supported. If there is no such link it should be removed.

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+releasePolicyLink: [https://www.jamf.com/resources/product-documentation/jamf-pro-release-notes/]
+eolColumn: true
+releaseDateColumn: true
+releaseCycleColumn: true
```
```suggestion
```

The `releaseCycleColumn` does not exist, see https://github.com/endoflife-date/endoflife.date/blob/master/CONTRIBUTING.md for more information about the schema.

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+    releaseDate: 2022-10-10
+    latest: "10.50.0"
+    eol: 2025-10-10
+    
```
```suggestion
```

No need for a blank line between `eol` and `link` (same for all releases).

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+> that helps organizations manage and secure their Apple devices.It provides a comprehensive
+> set of features for deploying and updating apps and software,configuring device settings,
+> enforcing security policies, and remotely wiping devices in the event of theft or loss.
+> Jamf Pro is a popular choice for organizations of all sizes, from small businesses to large enterprises.
```
```suggestion
> [Jamf-Pro](https://www.jamf.com/products/jamf-pro/) is a mobile device management (MDM) solution
> that helps organizations manage and secure their Apple devices. It provides a comprehensive
> set of features for deploying and updating apps and software, configuring device settings,
> enforcing security policies and remotely wiping devices in the event of theft or loss.
```

Removed the last sentence as it is not factual / can be seen as advertising.

marcwrobel commented on products/Jamfpro:
```diff
@@ -0,0 +1,52 @@
+> enforcing security policies, and remotely wiping devices in the event of theft or loss.
+> Jamf Pro is a popular choice for organizations of all sizes, from small businesses to large enterprises.
+
+Jamf Pro documentation: https://docs.jamf.com/jamf-pro/
```
Here you should summarize the release / support / EOL policy for Jamf Pro.

marcwrobel reviewed (changes requested):
Thanks for this contribution @EdwinThomaz !

I left you a few comments / suggestions. Do not hesitate if you need some help.

EdwinThomaz committed:
Update products/Jamfpro

- Removed Gap

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

EdwinThomaz committed:
Update products/Jamfpro

Removed "-"

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

EdwinThomaz committed:
Update products/Jamfpro

removed " iconSlug: jamf_pro"

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

EdwinThomaz committed:
Update products/Jamfpro

Removed "releaseCycleColumn: true"

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

EdwinThomaz committed:
Update products/Jamfpro

Removed GAP

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

EdwinThomaz committed:
Update products/Jamfpro

Commited suggested changes

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

EdwinThomaz committed:
Update Jamfpro

- Updated summary as suggested

EdwinThomaz committed:
Rename Jamfpro to Jamfpro.md

- Updated file name with .md extension

EdwinThomaz committed:
Update Jamfpro.md

- Removed "Space" Between eol and Link

marcwrobel committed:
Rename file

marcwrobel committed:
Normalize file

marcwrobel commented on products/jamf-pro.md:
```diff
@@ -0,0 +1,48 @@
+> of features for deploying and updating apps and software, configuring device settings, enforcing
+> security policies and remotely wiping devices in the event of theft or loss.
+
+Jamf Pro releases are typically announced in the fall and released in the winter. It follows an N-3
```
According to https://docs.jamf.com/10.28.0/jamf-pro/release-notes/Jamf_Pro_System_Requirements.html, it looks more like the support policy for Apple OS in Jamf Pro, not the Jamf Pro software support policy.

Can you provide a link that would back this information (it will also be used as the `releasePolicyLink`) ?

marcwrobel replied:
Reopen this conversation: @EdwinThomaz, https://docs.jamf.com/10.28.0/jamf-pro/release-notes/Jamf_Pro_System_Requirements.html does not explain what you documented in the description. Can you provide another official link that would back the description ?

marcwrobel replied:
Reopening again, https://learn.jamf.com/bundle/jamf-pro-release-notes-current/page/System_Requirements.html document the same thing as previous links. @EdwinThomaz this documentation is about the Apple OS support in Jamf Pro, it's not the Jamf Pro support policy.

EdwinThomaz replied:
-  Adding The replacement link
-  Support policy can be viewed for each individual version

marcwrobel replied:
@EdwinThomaz, I really do not see where I can confirm what you described on https://learn.jamf.com/bundle/jamf-pro-release-notes-10.50.0/page/Deprecations_and_Removals.html.

Where does it say that the previous 3 major versions are supported on https://learn.jamf.com/bundle/jamf-pro-release-notes-10.50.0/page/Deprecations_and_Removals.html or https://learn.jamf.com/bundle/jamf-pro-release-notes-current/page/System_Requirements.html ?

You also say that:
- _Jamf Pro releases are typically announced in the fall and released in the winter_
-  _The EOL date for each version of Jamf Pro is typically announced one year in advance._

Can you provide an example for each of these announcement in the comments ?

EdwinThomaz committed:
Update jamf-pro.md

- Committed suggested updates

EdwinThomaz committed:
Update jamf-pro.md

- Suggested changes made
- Updated releasepolicylink
- Updated link for description

EdwinThomaz committed:
Updated Support policy link 

- Updated support policy link
- Each versions with their support policy can be viewed with respective filtering
